tools/driver/p4c_src/main.py

```diff
@@ -369,17 +473,17 @@ def main():
               "ERROR: it seems that both (firstly) a JSON input pathname "
               'was specified via "' + JSON_input_flag + '"'
               " _and_ (secondly) " + str(len(opts.P4_source_files)) + ""
-              ###                                                     ^^
-              ### The above nonsense is to get rid of a W504 in “pycodestyle”
-              ###   [“W504 line break after binary operator”]
-              ### The same goes for “"" +” two lines down.
+                                                                               ###                                                     ^^
+                                                                               ### The above nonsense is to get rid of a W504 in “pycodestyle”
+                                                                               ###   [“W504 line break after binary operator”]
+                                                                               ### The same goes for “"" +” two lines down.
```

Comment:
This is not nice formatting :-(.

Reply:
Yes, will fix that. In general there is no good way to fix inline comments. I will try to move all comments that are inline to a separate line.

Reply:
It would be best if that could be specified in the style.